customers/schema.py

```diff
@@ -144,6 +172,98 @@ def get_node(cls, info, id):
             )
 
 
+class BoatCertificateInput:
+    file = Upload()
+    certificate_type = BoatCertificateTypeEnum()
+    valid_until = graphene.Date()
+    checked_at = graphene.Date()
+    checked_by = graphene.String()
+
+
+class CreateBoatCertificateMutation(graphene.ClientIDMutation):
+    class Input(BoatCertificateInput):
+        boat_id = graphene.ID(required=True)
+        certificate_type = BoatCertificateTypeEnum(required=True)
+
+    boat_certificate = graphene.Field(BoatCertificateNode)
+
+    @classmethod
+    @add_permission_required(BoatCertificate)
+    @transaction.atomic
+    def mutate_and_get_payload(cls, root, info, **input):
+        boat = get_node_from_global_id(info, input.pop("boat_id"), only_type=BoatNode)
```

Comment:
should this be `nullable=False`?

Reply:
ah, it will raise an error later on, ok, no need then 👍

Reply:
Ahh, but it actually should be `nullable=False`, good catch.